css/style.css

```diff
@@ -109,3 +111,181 @@ html {
     margin-left: 1rem;
   }
 }
+.hug {
+  text-align: center;
+}
+.hug__img {
+  height: 40px;
+  width: 40px;
+  left: 144px;
+  top: 4px;
+  border-radius: 32px;
+  padding: 8px;
+}
+
+.recent__work {
+  text-align: center;
+}
+.recent__work--text {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0em;
+  Fill: Solid;
+  color: #172B4D;
+  box-sizing: border-box;
+  height: 97px;
+  width: 327px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-content: center;
+  grid-gap: 16px;
+}
+
+.work {
+  border-radius: 8px;
+  margin: 20px;
+  border: 1px solid #D0D9D4;
+  height: 100%;
+}
+.work__img {
+  width: 100%;
+  height: 60%;
+  border-radius: 8px;
+}
+
+.work__content {
+  left: 0px;
+  top: 252px;
+  border-radius: 8px;
+  padding: 16px;
+  display: grid;
+  text-align: center;
+}
+
+.work__Title--text {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  fill: solid #3A4A42;
+}
+
+.work__done__group {
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+}
+.work__done__list {
+  list-style: none;
+  border: 1px solid #CCC;
+  background-color: #EBF0EE;
+  border: 4px;
+  font-family: Inter;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 8px 12px;
+  margin-top: 15px;
+}
+.work__done__link {
+  text-decoration: none;
+  color: #3A4A42;
+}
+
+.btn__style {
+  display: grid;
+  justify-content: center;
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.btn {
+  height: 48px;
+  width: 122px;
+  border-radius: 4px;
+  padding: 12px;
+  background-color: #36B37F;
+  color: #FFFFFF;
+  cursor: pointer;
+  border: none;
+  font-size: 17px;
+}
+
+.Abtn {
+  width: 159px;
+  margin: 12px;
+}
+
+.btn:hover {
+  background-color: #36B37F;
+}
+
+.button:active {
+  background-color: #3e8e41;
+  transform: translateY(4px);
+}
+
+.about__me__Container {
+  width: 100%;
+  margin-top: 10rem;
+  margin-bottom: 30px;
+}
+
+.about__me {
+  text-align: center;
+}
+.about__me__title {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0em;
+  Fill: Solid;
+  color: #172B4D;
+  box-sizing: border-box;
+}
+.about__me__text {
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  margin: 0;
+}
+.about__me__paragraph {
+  font-family: Inter;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0em;
+  text-align: center;
+  color: #42526E;
+  margin: 10px;
+}
+
+.bgrht {
+  position: relative;
+}
+
+.bgrht__img {
+  width: 68px;
+  height: 127px;
+  position: absolute;
+  top: -45px;
+  right: 0;
+}
```

Comment:
- Please, do fix the issue with the image, as it is not showing to improve your design.